docs/resources/data_source_config_lbac_rules.md

```diff
@@ -0,0 +1,77 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_data_source_config_lbac_rules Resource - terraform-provider-grafana"
+subcategory: "Grafana Enterprise"
+description: |-
+  Manages LBAC rules for a data source.
+  !> Warning: The resource is experimental and will be subject to change. This resource manages the entire LBAC rules tree, and will overwrite any existing rules.
+  Official documentation https://grafana.com/docs/grafana/latest/administration/data-source-management/teamlbac/HTTP API https://grafana.com/docs/grafana/latest/developers/http_api/datasource_lbac_rules/
+  This resource requires Grafana >=11.4.0.
```

Comment:
I think we actually need to set it to 11.5.0 - 11.4.0 will not include the changes that we need ([see this internal thread](https://raintank-corp.slack.com/archives/C010WU9Q5TJ/p1732112639018819)).